atlp-rwanda/trojans-ec-fe
# #184760060 - Cart Operations

### What does this PR do?

Adds the adding to  cart ,viewing it , clearing the whole cart  and clearing a single cart item and updating quantity both ui and functionalities

### Description of Task to be completed?
Adds the adding to  cart ,viewing it , clearing the whole cart  and clearing a single cart item and updating quantity basically all cart operations

### How should this be manually tested?

Mainly on the deployed site/PR preview and npm test
PS: You have to be logged in to see the products
One of the user one can use testBuyer@example.com with password of default

The preview is available done below


### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/n/projects/2628461/stories/184760057
https://www.pivotaltracker.com/n/projects/2628461/stories/184760058
https://www.pivotaltracker.com/n/projects/2628461/stories/184760059
https://www.pivotaltracker.com/n/projects/2628461/stories/184760060

### Screenshots (if appropriate)
### Figma
#### Cart view
![figma](https://user-images.githubusercontent.com/111729060/233549064-1188a534-1b8b-4b16-90db-9a8be6501098.PNG)


### Actual
#### Cart view
![cart](https://user-images.githubusercontent.com/111729060/233548662-a2345406-90d4-4acb-9957-514ba92d11f9.PNG)
